### PR TITLE
fix(connector): Update mandate PMT configs for Mandate Supported Connectors

### PR DESCRIPTION
**Pull request by awasthi21**
## Type of Change

- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement
- [ ] Refactoring
- [x] Dependency updates
- [ ] Documentation
- [ ] CI/CD

### Additional Changes

- [ ] This PR modifies the API contract
- [ ] This PR modifies the database schema
- [x] This PR modifies application configuration/environment variables

## Checklist

- [x] I formatted the code `cargo +nightly fmt --all`
- [x] I addressed lints thrown by `cargo clippy`
- [x] I reviewed the submitted code
- [ ] I added unit tests for my changes where possible

### CODE REVIEW

**Review by deepanshu-iiitu — APPROVED** [review]

File: config/development.toml
Comment (SanchithHegde) [inline config/development.toml:1]: Nit: Can remove the comments here. Optional change.

File: config/deployments/sandbox.toml
Comment (SanchithHegde) [inline config/deployments/sandbox.toml:1]: Can we make the same changes in `integration_test.toml` file as well?

File: config/development.toml
Comment (awasthi21) [reply config/development.toml:1]: removed

File: config/deployments/sandbox.toml
Comment (awasthi21) [reply config/deployments/sandbox.toml:1]: yes added

**Review by SanchithHegde — APPROVED** [review]

**Review by deepanshu-iiitu — APPROVED** [review]

**Pull request merged**